sdk/src/main/scala/com/horizen/account/mempool/MempoolMap.scala

```diff
@@ -149,8 +142,10 @@ class MempoolMap(stateReader: AccountStateReader) extends ScorexLogging {
    */
   def takeExecutableTxs(limit: Int): Iterable[SidechainTypes#SCAT] = {
```

Comment:
Simple case: we have 1M txs in the mempool
Forger will take all 1M sorted elements and than will take only the first ~1400 valid txs.

An improvement here can be to have a lazy data structure.
We may put it into backlog

Reply:
Ok, probably I will see it in the recurrent PR https://github.com/HorizenOfficial/Sidechains-SDK/pull/593

Reply:
Correct, this issue was addressed in PR #593